### PR TITLE
Remove pcntl check

### PR DESCRIPTION
**Pull request by dreua**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [x] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: protected/humhub/libs/SelfTest.php [hunk 1]
```diff
@@ -160,9 +160,10 @@ public static function getResults()
         }
 
         // Checks PCNTL Extension
-        $title = 'PHP - PCNTL Extension';
-
-        if (function_exists('pcntl_signal')) {
+	$title = 'PHP - PCNTL Extension';
+	// PCNTL may be disabled in webserver context but available in php-cli
+	// where it is actually needed.
+        if (function_exists('pcntl_signal') || shell_exec('php -r \'echo function_exists("pcntl_signal");\'') == '1') {
```
Comment (ArchBlood) [inline protected/humhub/libs/SelfTest.php:166]: To be honest `shell_exec()` can't be used by all systems in some shared hosts so I'd recommend first doing an `ini_get()` to check if permissions allow for it to be used.

Comment (dreua) [reply protected/humhub/libs/SelfTest.php:166]: Yeah, I was wondering whether that would be a problem, not sure how to reproduce and test this though: try/catch / ini_get / function_exists('shell_exec')? Maybe a small function with try/catch could be the most solid solution, what do you think?

Comment (ArchBlood) [reply protected/humhub/libs/SelfTest.php:166]: It would be more logical to generate a `try` and then `catch` for unexpected results that could happen

Comment (ArchBlood) [reply protected/humhub/libs/SelfTest.php:166]: As to check `ini_get('disable_functions')` should do to display disabled functions but not sure if this is too specific or not 🤔

Comment (luke-) [reply protected/humhub/libs/SelfTest.php:166]: Checking whether 'shell_exec' is available is apparently a bit more complicated.
https://stackoverflow.com/questions/21581560/php-how-to-know-if-server-allows-shell-exec

Important would be that no fatal error is triggered if the function is not available.

**Comment by luke-** [comment]
Maybe we should simply remove the whole check. Especially in the web context the PCNTL extension should not be active.

**Comment by dreua** [comment]
That would be fine with me, too. Otherwise I'd try to update this PR with a try/catch solution this weekend, let me know if I should do that.

**Comment by luke-** [comment]
@dreua Please just remove the check. I think that causes too much confusion.

**Comment by ArchBlood** [comment]
I can also agree, as it stands if you only have a check for `shell_exec()` then you'd also have to check all other alternatives to `shell_exec()` 🤔

**Comment by dreua** [comment]
I removed the check, as requested. Let me know if you want a different entry in the changelog, I just tried to fit in with the style.

**Review by luke- — APPROVED** [review]

**Comment by luke-** [comment]
@dreua Great, thank you!

**Pull request merged**